README.md

```diff
@@ -1,31 +1,141 @@
-micropython-lib
-===============
+# micropython-lib
 
-This is a repository of libraries designed to be useful for writing
-MicroPython applications.
+This is a library of packages designed to be useful for writing MicroPython
```

Comment:
maybe continue to use the word "repository" to make certain the word is not confused with package/module/library?

Reply:
I see you clarified the terminology below... but still it might be worth using the word "repository" here?